packages/braid-design-system/src/lib/components/Button/Button.docs.tsx

```diff
@@ -45,11 +63,11 @@ const choosingRightButtonDoc = [
             Default is a <Strong>neutral</Strong> tone and{' '}
             <Strong>ghost</Strong> variant:
```

Comment:
```suggestion
            <Strong>ghost</Strong> variant
```
Wonder if we can drop the colons in the examples here for consistency?